samples/snippets/csharp/tour/statements/Program.cs

```diff
@@ -75,10 +75,10 @@ static void DoStatement(string[] args)
             do 
             {
                 s = Console.ReadLine();
-                if (string.IsNullOrEmpty(s)) 
-                    Console.WriteLine(s);
+                Console.WriteLine(s);
```

Comment:
I think the intention was:

* If the string is not null or empty, print it out and repeat.
* If the string is null or empty, end the loop.

To do that, the `if` should have been changed by adding a `!`, not by removing it completely.
In other words, I think the code should be:

```c#
string s;
do 
{
    s = Console.ReadLine();
    if (!string.IsNullOrEmpty(s)) 
        Console.WriteLine(s);
} while (!string.IsNullOrEmpty(s));
```

Reply:
Sorry, I didn't realize I was too slow. I guess it's fine this way too.

Reply:
That's what I thought. I will print a blank line, which is just fine.

Reply:
Yes, but I thought printing an empty line is not a big problem and less repeat makes me feel better :P